website/static/js/pages/project-base-page.js

```diff
@@ -32,4 +32,41 @@ $(document).ready(function() {
     $.getJSON(node.urls.api, function(data) {    
         $('body').trigger('nodeLoad', data);
     });
+
+    var self = this;
+    var thresholdScrollPosition = 50;
+    var smallScreenSize = 767;
+    var nonNavTopMargin = 50;
+    var navMaxTopMargin = 95;
+    self.checkPanelPosition = function() {
```

Comment:
`adjustPanelPosition` might be a better name because it conveys that this does some manipulation of the DOM